r2/r2/public/static/js/expando-md.js

```diff
@@ -63,6 +63,10 @@
       }
 
       // Videos
+      check = getVideoExtension($thing.attr('href'));
+      if (check) {
+        $thing.after(' <a class="' + buttonClass + ' ' + buttonClosedClass + '" data-type="video" data-expando-exists="false" href="javascript:void(0);">' + check + '</a> ');
```

Comment:
you have to return here, it's doing an extraneous YT check

Reply:
Whoops! I think I forgot a `break;` somewhere too.